PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,10 @@
-Fixes #(issue)
+Fixes #
 
 # Done is when (DoD):
-- [ ] Modifications within components are reflected by tests.
-- [ ] A self-review of the code has been performed.
-- [ ] The modified component properties have been added to the typescript typings.
-- [ ] Potential design changes have been reviewed by a designer.
-- [ ] The modifications are well documented (README.md, etc.) and showcased in the stories.
-- [ ] The modifications are documented in the code, particularly in hard-to-understand areas (focus on **WHY**).
-- [ ] The modifications are shortly added to CHANGELOG.md with the issue number.
-- [ ] The modifications still work in IE.
-- [ ] The modifications are according to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)
+- [ ] Code selfreview has been performed.
+- [ ] Tests are sufficient.
+- [ ] Modifications still work in IE.
+- [ ] Documentation is up to date.
+- [ ] Typescript typings for properties are up to date.
+- [ ] Designers approved changes or no approval needed.
+- [ ] Modifications follow our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)
```

Comment:
You left Best Pract.? We do not need that because we can not forget this point. But "adding a changelog" would be useful because we have forgotten that from time to time.

Reply:
I like to add changelogs AFTER I merged somehow, because you only know the version when you actually publish. I have that problem on another PR now, where I need to change the changelog again. So I prefer to first publish and then add whatever was really happening.

Reply:
And no? Best practices are the last point. Or what do you mean?

Reply:
True that. Changelogs are missing. We discussed as a team, that changelogs must be adde to the pr.